.github/workflows/ci.yml

```diff
@@ -54,6 +54,27 @@ jobs:
       - name: check reminders and notify
         uses: agrc/reminder-action@v1
 
+  lockfiles:
+    runs-on: ubuntu-latest
+    name: "Lockfiles"
+    env: 
+      TARGET_DOCS_LOCKFILE_PATH: .lockfiles/py310-dev.lock
+      DOCS_LOCKFILE_PATH: /tmp/lockfile
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        id: setup-python
+        with:
+          python-version: '3.10'
+      - name:  Verify lockfile integrity 
+        run: |
+          git fetch origin main:main
+          if git diff --name-only origin/main -- pyproject.toml | grep -q pyproject.toml; then
+            echo "pyproject.toml has been modified compared to the main branch."
+            echo "Lockfile needs to be updated."
```

Comment:
According to your comment, shouldn't this be "Lockfile might need to be updated"?